package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "nodejs-st-22-rest-api",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
```

Comment:
and again the `ghost` did all the job...

Reply:
Done.